terraform-module/modules/frontend-github-deployment-env/README.md

```diff
@@ -0,0 +1,61 @@
+# GitHub Deployment Environment
+
+Creates a GitHub deployment environment, and injects provided AWS credentials
+into that new environment's secrets. See GitHub documentation on
+[GitHub deployment environments](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment).
+
+## How to use
+
+```hcl
+module "deployment_env" {
+  source = "api.env0.com/ed3d7d49-aa1a-4456-8ccb-3b56dd0e7070/github-deployment-env/aws"
```

Comment:
I'm not sure I understand how this works, will this module still be available in the env0 repository?

Initially I thought we would create a separate terraform module from the existing one.

Reply:
this readme needs to be updated, you are right. it will probably look like that
  source = "git@github.com:pleo-io/spa-tools.git//terraform-module/modules/frontend-github-deployment-env?ref=terraform-module-v3.1.1"

i need to test it before being 100% sure though. I will change the readme and get back to it when i tested from the tf repo!